drivers/flash/flash_ite_it8xxx2.c

```diff
@@ -522,15 +474,13 @@ static int flash_it8xxx2_init(const struct device *dev)
 	/* Initialize mutex for flash controller */
 	k_sem_init(&data->sem, 1, 1);
 
-	flash_code_static_cache(dev);
-
 	return 0;
 }
 
 #if defined(CONFIG_FLASH_PAGE_LAYOUT)
 static const struct flash_pages_layout dev_layout = {
-	.pages_count = DT_REG_SIZE(SOC_NV_FLASH_NODE) /
-			DT_PROP(SOC_NV_FLASH_NODE, erase_block_size),
+	.pages_count =
+		DT_REG_SIZE(SOC_NV_FLASH_NODE) / DT_PROP(SOC_NV_FLASH_NODE, erase_block_size),
```

Comment:
Are these just cosmetic changes? These seem to be applied here in random places in the same commit

Reply:
That was probably auto-formatting. I've reverted the non-functional chnages.